src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```
@@ -9,6 +10,54 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+
+        if (user == null) {
+            throw new UserNotRegisterException("User is null.");
+        }
+
+        if (user.getLogin() == null) {
+            throw new UserNotRegisterException("Login user's is null.");
+        }
+
+        if (user.getPassword() == null) {
+            throw new UserNotRegisterException("Password user's is null.");
+        }
+
+        if (user.getAge() == null) {
+            throw new UserNotRegisterException("Age user`s is null.");
+        }
+
+        if (user.getLogin().length() == 0) {
+            throw new UserNotRegisterException("Login user's is empty string.");
+        }
+
+        if (user.getPassword().length() == 0) {
+            throw new UserNotRegisterException("\"Password user's is empty string.");
+        }
+
+        if (user.getAge() == 0) {
+            return null;
```

Issue: According to the checklist, you should throw a UserNotRegisterException if the user's age is 0 (invalid), not return null.